eng/pipelines/rolling-internal-validation-pipeline.yml

```diff
@@ -72,7 +72,17 @@ extends:
         suppressionFile: $(Build.SourcesDirectory)/.config/guardian/.gdnsuppress
       tsa:
         enabled: ${{ not(parameters.disableTSA) }}
-        configFile: $(Build.SourcesDirectory)/.config/tsa/tsaoptions.json
+        config:
+          codebaseName: 'Go'
+          notificationAliases:
+            - 'golangdev@microsoft.com'
+          codebaseAdmins:
+            - 'redmond\golangdev'
+          instanceUrl: 'https://devdiv.visualstudio.com/'
+          projectName: 'DEVDIV'
+          areaPath: 'DevDiv\GoLang'
+          iterationPath: 'DevDiv'
+          allTools: true
```

Comment:
Maybe we should have a separate pipeline for go-mirror after all... it's a little odd now that this actually replaces go-infra's `tsaoptions.json` too (but only for this particular purpose). Filed https://github.com/microsoft/go-lab/issues/230.